rpm/SPECS/policycoreutils.spec

```diff
@@ -0,0 +1,31 @@
+Name:           policycoreutils
+Version:        3.2
+Release:        1%{?dist}
+Group:          System Environment/Base
+Summary:        policycoreutils package
+License:        GPLv2
```

Comment:
We need policycoreutils rpm while building as well as while installing the docker-ce binaries.
Since the version required is not available in repo, we have to build it from source.